.github/workflows/stress-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Stress Tests
+
+on: [push]
+
+jobs:
+  stress-tests:
+    runs-on: ubuntu-latest-4-cores
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - name: Set up Node.js
+        uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e
+        with:
+          node-version: 22
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build storybook
+        run: npx storybook build
+        working-directory: packages/react
+      - name: Run storybook
+        id: storybook
+        run: |
+          npx serve -l 6006 packages/react/storybook-static &
+          pid=$!
+          echo "pid=$pid" >> $GITHUB_OUTPUT
+          sleep 5
+      - name: Run Stress Tests
+        uses: docker://mcr.microsoft.com/playwright:v1.51.0-jammy
+        env:
+          STORYBOOK_URL: 'http://172.17.0.1:6006'
+        with:
+          args: npx playwright test --grep @stress-test"
+      - name: Stop storybook
+        if: ${{ always() }}
+        run: kill ${{ steps.storybook.outputs.pid }}
+      - name: Download previous benchmark data (if any)
+        uses: actions/cache@v4
+        with:
+          path: ./cache
+          key: stress-tests-benchmark
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          tool: 'customSmallerIsBetter'
+          output-file-path: results.json
+          # Where the previous data file is stored
+          external-data-json-path: ./cache/stress-tests-benchmark-data.json
+          # Workflow will fail when an alert happens
+          fail-on-alert: true
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          alert-threshold: '150%'
+          # Enable alert commit comment
+          comment-on-alert: true
+          # Mention @rhysd in the commit comment
+          alert-comment-cc-users: '@hectahertz'
```

Comment:
are you sure you want to be CC'd in every comment? 😅

Reply:
Hahaha, yes for now, The plan is to remove it once we've battle tested this enough